projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -5428,16 +5429,14 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         this.endRowTransaction(commit, row);
 
         if (event) {
-            const currentCell = this.gridAPI.get_cell_by_index(ri, columnIndex);
-            if (currentCell) {
+            if (cell) {
+                const currentCell = this.gridAPI.get_cell_by_index(ri, columnIndex);
```

Comment:
Now when the cell in edit mode is not visible and click on Done button an error is returned.
There should be check:  if (currentCell) and then to focus it.

Reply:
thank you for this catch.
should be fixed now